src/module/actor/base.ts

```diff
@@ -567,6 +567,11 @@ class ActorPF2e extends Actor<TokenDocumentPF2e, ItemTypeMap> {
 
         super.prepareData();
 
+        // Call post-derived-preparation `RuleElement` hooks
+        for (const rule of this.rules) {
+            rule.afterPrepareData?.();
```

Comment:
try catch block is no longer needed?

Reply:
REs used to be a lot dirtier, rarely checking the the data they received.